src/brouwers/shop/models/presentation.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from django.utils.translation import ugettext_lazy as _
+
+
+@python_2_unicode_compatible
+class HomepageCategory(models.Model):
+    main_category = models.ForeignKey('Category', related_name='homepage_categories', null=True, blank=True)
```

Comment:
doesn't seem logical to me that this field can be left empty? So drop the `null=True` and `blank=True`, and I'd even make it `unique=True`. So... should in essence be a `OneToOneField` probably